curator-framework/src/main/java/org/apache/curator/framework/api/RemoveWatchesBuilder.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.curator.framework.api;

```

Comment:
I think quietly() is a good idea as its explicit. Also, guaranteed() will never tell the user that the remove has failed, whereas quietly would indicate failure (except for NoWatcher exceptions).